### ISSUE TITLE
Unable to add namespaces to lightning component markup

### ISSUE THREAD

**Issue by Cassius-Bardsley** [joeferraro/MavensMate#913]
* **MavensMate Desktop Version**: v0.0.11-beta.7

* **Editor**: Sublime

* **MavensMate Plugin Version**: 7.0.2

* **Platform/Version**: OS X 10.12.3

* **Proxy Details**: N/A

* **Salesforce API Version**: 26

* **Salesforce Server**: NA35

I have enabled a namespace in my org, and am attempting to add markup to lightning components per the [Namespace Usage Examples and Reference][1] documentation. The component saves successfully in MavensMate/Sublime, however if I refresh from server, or view the component in Developer Console, I see that the namespace that I just added is missing. 

**Steps to reproduce:**

In an org with a set namespace prefix, create a lightning component and an apex class. 

Using MavensMate and Sublime, edit the first line of a lightning component as follows:

*Before:*

    <aura:component controller="MyController">

*After:*

    <aura:component controller="MyNamespace.MyController">

Save and observe that you receive a "successful" message.

Open the component in the Developer Console

*Expected:*

    <aura:component controller="MyNamespace.MyController">

*Actual:*

    <aura:component controller="MyController">

This also happens when attempting to add a namespace prefix to a component tag such as `<c:myComponent/>` -> `<MyNamespace:myComponent/>` or static resource references such as `{!$Resource.resourceName}` -> `{!$Resource.MyNamespace__resourceName}`

  [1]: https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/namespace_using_reference.htm

**Comment by kidtsunami** [comment]
@Cassius-Bardsley when you add the namespace using the Developer Console, does it stay there?

**Comment by Cassius-Bardsley** [comment]
@kidtsunami Yes. The namespace stays there when I add it using the dev console.

**Comment by kidtsunami** [comment]
@Cassius-Bardsley can you share the xml associated with the files in question?

**Comment by Cassius-Bardsley** [comment]
@kidtsunami Here is the contents of the cmp-meta.xml file associated with the lightning component bundle I am attempting to modify:

```<?xml version="1.0" encoding="UTF-8"?>
<AuraDefinitionBundle xmlns="http://soap.sforce.com/2006/04/metadata">
    <apiVersion>36.0</apiVersion>
    <description>A Lightning Component Bundle</description>
</AuraDefinitionBundle>
```
